deploy_widget.sh

```diff
@@ -10,6 +10,5 @@ mkdir -p $deploy_dir/../vendor/d3-3.4.13/
 
 cp dist/arethusa.min.js $deploy_dir
 cp dist/arethusa.min.map $deploy_dir
-cp dist/arethusa_packages.min.js $deploy_dir/arethusa.packages.min.js
```

Comment:
This is a much better flow than than running `git checkout` in the other repository every time. Thanks!